File: java/oop/InterfaceMan-App/src/com/techlabs/imannerable/Boy.java
package com.techlabs.imannerable;

public class Boy implements Imanable,Iemotionable {

	@Override
	public void laugh() {
		System.out.println("Boy laughs.....");
		
	}

	@Override
	public void wish() {
	System.out.println("Boy wishes.......");
		
	}

	@Override
	public void depart() {
		System.out.println("Boy departs.....");
		
	}

	@Override
	public void cry() {
		System.out.println("Boys cry..........");
		
	}

}

File: dotNet/csharp/InterfaceManApp/InterfaceManApp/Boy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace InterfaceManApp
{
    class Boy : Imannable, Iemotionable
    {
        public void Cry()
        {
            Console.WriteLine("Boy cry...........");
        }

        public void Depart()
        {
            Console.WriteLine("Boy departs...........");
        }

        public void Laugh()
        {
            Console.WriteLine("Boy Laugh...........");
        }

        public void Wish()
        {
            Console.WriteLine("Boy Wishes...........");
        }
    }
}
